lib/assets/javascripts/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "turbograft",
+  "version": "1.0.0",
+  "description": "Turbograft extends [Turbolinks](https://github.com/rails/turbolinks), allowing you to perform partial page refreshes.",
+  "scripts": {
+    "test": "dev test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/turbograft.git"
+  },
+  "author": "Shopify",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Shopify/turbograft/issues"
+  },
+  "homepage": "https://github.com/Shopify/turbograft#readme",
+  "dependencies": {
+    "promise-polyfill": "6.0.1"
```

Comment:
packages including a polyfill is an antipattern, the project that uses the package should decide what is needed for its target. the shopify admin already contains a promise polyfill for example